src/Microsoft.VisualStudio.Editors/PropPages/PackagePropPage.vb

```diff
@@ -230,7 +285,113 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
         ''' </summary>
         Private Sub NeutralLanguageComboBox_DropDown(sender As Object, e As EventArgs) Handles NeutralLanguageComboBox.DropDown
             PopulateNeutralLanguageComboBox(NeutralLanguageComboBox)
-            Common.SetComboBoxDropdownWidth(NeutralLanguageComboBox)
+            SetComboBoxDropdownWidth(NeutralLanguageComboBox)
+        End Sub
+
+        ''' <summary>
+        '''  Set the radio button selection and update both textboxes
+        ''' </summary>
+        ''' <param name="setLicenseExpression">Sets the radio button for LicensesExpression, if false sets to LicenseFile</param>
+        Private Sub SetLicenseRadioButtons(setLicenseExpression As Boolean)
+            If (Not _newLicensePropertyDetectedAtInit) Then
+                LicenseTypeFirstSelected()
+                _newLicensePropertyDetectedAtInit = False
+            End If
+            LicenseFileRadioButton.Checked = Not setLicenseExpression
+            LicenseExpressionRadioButton.Checked = setLicenseExpression
+            PackageLicenseExpression.Enabled = setLicenseExpression
+            LicenseFileNameTextBox.Enabled = Not setLicenseExpression
+            LicenseBrowseButton.Enabled = Not setLicenseExpression
+            If setLicenseExpression AndAlso Not LicenseFileNameTextBox.Text = "" Then
+                SetCommonPropertyValue(GetPropertyDescriptor("PackageLicenseFile"), "")
+                LicenseFileNameTextBox.Text = ""
+            ElseIf (Not PackageLicenseExpression.Text = "") Then
+                PackageLicenseExpression.Text = ""
+            End If
+        End Sub
+
+        Private Sub LicenseTypeFirstSelected()
+            'When the project has neither of the new license properties AND it has the license URL property and a new license type is selected
+            If (_licenseUrlDetected) Then
+                LicenseLineLabel.BackColor = Drawing.SystemColors.ControlDark
```

Comment:
Why are you seeing the color?

Reply:
Same reason as above, I am using that label as the separator and in the case that there is a LicenseUrl, it will display a warning.

Reply:
This will not play nicely with screen readers - you are using color to hide the warning?